### PR TITLE
integration tests for resource limitation

### PR DESCRIPTION
**Pull request by chanwit**
Add resource limitation tests.
Subtask of #888 

Signed-off-by: Chanwit Kaewkasi <chanwit@gmail.com>

### CODE REVIEW

**Comment by chanwit** [comment]
@aluzzardi PTAL

File: test/integration/resource_management.bats [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+function teardown() {
+	swarm_manage_cleanup
+	stop_docker
+}
+
+@test "resource limitation: memory" {
+	start_docker_with_busybox 1
+	swarm_manage
+
+	run docker_swarm run -m 48G busybox sh
```
Comment (aluzzardi) [inline test/integration/resource_management.bats]: Can you put something **crazy** ? :) Like 1000G?

**Comment by aluzzardi** [comment]
I think @vieux added a similar test in `api/run.bats`.

This one is better - could you remove the other one?

**Comment by vieux** [comment]
@chanwit can you please also move `@test "docker run with resources"` in this file and add a `docker info` after the test to make sure the docker info reports the right message.

**Comment by chanwit** [comment]
@vieux @aluzzardi PTAL

It seems `janky` failed because its clean up process (`docker rmi` called without a parameter).

**Comment by aluzzardi** [comment]
@chanwit I believe it's not because of the `rmi` :/

>    docker run87/87Build timed out (after 60 minutes). Marking the build as failed.

It looks like it's timing out. I'm relaunching this to see if it was transient

**Comment by aluzzardi** [comment]
Okay it worked

LGTM

**Comment by chanwit** [comment]
phew

**Pull request merged**